hamilton_launch_server/dockerize.sh

```diff
@@ -82,8 +132,14 @@ elif [ "$ACTION" == "build" ]; then
     else
         echo "ERROR: Unknown target platform [ lin | osx | win | rpi ]"
     fi
+elif [ "$ACTION" == "ffserver" ]; then
+    run_ffserver
+elif [ "$ACTION" == "ffmpeg" ]; then
+    run_ffmpeg
 else
-    echo "usage: $0 init "
+    echo "usage: $0 init [ server | livestream ]"
     echo "       $0 build [ lin | osx | win | rpi ]"
+    echo "       $0 ffserver"
+    echo "       $0 ffmpeg"
```

Comment:
I think these should be subsets of a run command.

Reply:
Or even better, just do `run streams` and it'll run both.

Reply:
You can't. I told you before, but the ffserver command cannot be run using & or detached either. You need to run it on a separate terminal. 

ffserver is just a hard to deal with command I think, which is why ffmpeg project deprecated it.

Reply:
aightttt